.github/workflows/pr.yml

```diff
@@ -24,35 +24,74 @@ jobs:
       - run: ls ./contract
       - run: yarn test
   integration:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-16core
+
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
-      - run: corepack enable
-      - run: yarn install
-      - run: yarn start:docker
-      - run: yarn build
-      # Borrowed from https://github.com/DCFoundation/cosmos-proposal-builder/blob/main/.github/workflows/pr.yml#L43-L61
-      - name: Wait for Docker container to be ready
+
+      - name: Enable Corepack
+        run: corepack enable
+
+      - name: Install deps
+        run: yarn install
+
+      - name: Clone and build agoric-sdk
+        run: |
+          git clone https://github.com/Agoric/agoric-sdk.git
+          cd agoric-sdk
+          yarn install
+          yarn build
+          yarn link-cli
+          echo "$HOME/bin" >> $GITHUB_PATH
+
+      - name: Build dapp-agoric-basics
+        run: yarn build
```

Comment:
Can remove, unless we are interested in testing the UI

Reply:
Do we plan on testing UI interactions in our pipeline, or just rely on `unit`?